dynamic.js

```diff
@@ -96,9 +96,12 @@ module.exports = function (fastify, opts, next) {
       const schema = transform
         ? transform(route.schema)
         : route.schema
-      const path = route.url.startsWith(basePath)
+      let path = route.url.startsWith(basePath)
         ? route.url.replace(basePath, '')
         : route.url
+      if (!path.startsWith('/')) {
+        path = '/' + path
```

Comment:
I think this is correct, but do we know who is removing this slash?

Reply:
Yes, the line right above with
```js
route.url.replace(basePath, '')
```
if `basePath` is defined as `/`, then it will be removed everywhere.